test/functional/p2p-fullblocktest.py

```diff
@@ -18,6 +18,7 @@
 import time
 from test_framework.key import CECKey
 from test_framework.script import *
+from test_framework.script import hash160
```

Comment:
It seems a little strange that hash160 is in script.py and hash256/sha256 are in mininode.py can we put all three in one place (probably mininode.py)? You're changing most of the places where this is being imported (smartfees.py and test_framework/address.py seem to be the only other ones). It might be worth just changing this, too.